app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -180,6 +180,7 @@ async def _merge_intensity_and_date_conf(
         source_type=RasterSourceType.raster,
         source_driver=RasterDrivers.geotiff,
         source_uri=[date_conf_uri, intensity_uri],
+        calc="[((A -((A>=30000) * 10000) - ((A>=20000) * 20000)) * (A>=20000)/255).astype('uint8'), ((A -((A>=30000) * 10000) - ((A>=20000) * 20000)) * (A>=20000) % 255).astype('uint8'), (((A>=30000) + 1)*100 + B).astype('uint8')]",  # this will not be used in PixETL but pydantic requires some input value
```

Comment:
Why does Pydantic require something other than None? Isn't calc Optional?
In any case, if it is ignored I think it might be less confusing if we put in something more clearly place-holder like "This calc line does nothing" or something. Despite the comment, it's at the end of a long line and might not be seen immediately.

Reply:
With my latest changes I enforce `calc` to be not optional when there are more than two input source uri, b/c normally pixetl would merge the two layers into one.

In this case, we don't use the creation options with pixetl but with the custom `build_rgb` script. I added the formular b/c this would be the numpy function we would have to use if we would use pixetl for this. But this would mean that pixetl could create output files with more than one band. and we are not quite there yet.

I am fine to put some other string here and leave it as a comment, just in case we need the formula later.